code/drasil-printers/lib/Language/Drasil/TeX/Print.hs

```diff
@@ -76,10 +76,22 @@ lo (Graph ps w h c l)    _  = toText $ makeGraph
 lo (Cell _) _               = empty
 lo (CodeBlock _) _          = empty
 
+-- | Helper for converting layout objects into a more printable form.
+-- This function is specific to definitions.
+lo' :: LayoutObj -> PrintingInformation -> D
+lo' (HDiv _ con _)      sm = print' sm con
+lo' (EqnBlock contents) _  = makeEquation contents 0.8
+lo' obj sm                 = lo obj sm
```

Comment:
Minor nitpick for alignment 😅

```suggestion
lo' obj                 sm = lo obj sm
```

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3910/commits/f78f58a2d6d93b74e7b82afe12e536ffedbacf8c.